lib/tasks/brakeman.rake

```diff
@@ -0,0 +1,5 @@
+task :brakeman do
+  sh <<end
+mkdir -p tmp && (brakeman --quiet --output tmp/brakeman.out --exit-on-warn && echo "no warnings or errors") || (cat tmp/brakeman.out; exit 1)
+end
```

Comment:
Is there an extra 'end' here?

Reply:
No, the first end is the closure to `<<end`, the second is the closure to the task itself 😄